be/btree.c

```diff
@@ -476,9 +489,11 @@ static void be_btree_split_child(struct m0_be_btree *btree,
 	i = 0;
 	while (i < new_child->bt_num_active_key) {
 		new_child->bt_kv_arr[i] = child->bt_kv_arr[i + BTREE_FAN_OUT];
+		new_child->bt_ik[i] = child->bt_ik[i + BTREE_FAN_OUT];
```

Comment:
This will not work. You need to dereference values from child->bt_ik which correspond to child->bt_kv_arr[i + BTREE_FAN_OUT].